lib/util/git/behind-base-branch-cache.ts

```diff
@@ -0,0 +1,18 @@
+import { getCache } from '../cache/repository';
+
+export function getCachedBehindBaseResult(
```

Comment:
Document/describe this function. It's not intuitive enough right now, especially with the ! And === combined logic at the end